app/src/main/java/com/example/androidseminar/util/room/UserLoginData.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.androidseminar.util.room
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class UserLoginData(
+    var id:String,
+    var password:String,
+    var autoSelected:Boolean
+){
+    @PrimaryKey(autoGenerate = true) var key:Int=0
+}
```

Comment:
```suggestion
@Entity
data class UserLoginData(
    @PrimaryKey(autoGenerate = true) var key:Int=0,
    var id:String,
    var password:String,
    var autoSelected:Boolean
)
```

이렇게 묶을 수 있습니다! 그리고 Entity의 경우 불변객체(val)로 두는것이 좋을 것 같습니다.

Reply:
오홍 저렇게 묶는게 더 깔끔하고 직관적이군요